docs/install-docker-compose-quickstart.md

```diff
@@ -0,0 +1,105 @@
+# Docker Quickstart
+
+[Docker](https://www.docker.com/) is the quickest way to download and start developing with Nakama. By using Docker you are able to:
+
+* Install to a pristine environment
+* Easily install and run the [CockroachDB](https://www.cockroachlabs.com/) or [PostgreSQL](https://www.postgresql.org/) database for Nakama
+* Take snapshots, remove, and re-install Nakama without affecting your primary operating system
+* Enjoy a quick and simplified installation experience regardless of your OS
+
+Following this guide, you will use [Docker Compose](https://docs.docker.com/compose/) to quickly and easily define all the necessary services and run your local developement instance of Nakama.
+
+## Prerequisites
+
+Before proceeding ensure that you have [installed Docker Desktop](https://docs.docker.com/get-docker/).
+
+## Running Nakama
+
+1. Start by creating a directory where your Nakama server will sit, for example `Desktop/nakama`.
+2. In this folder create a `docker-compose.yml` file and open it using your preferred text editor.
+3. Heroic Labs provides two YML files for use: using either [CockroachDB](https://github.com/heroiclabs/nakama/blob/master/docker-compose.yml) or [PostgreSQL](https://github.com/heroiclabs/nakama/blob/master/docker-compose-postgres.yml) as the database.
+Copy and paste the contents of your preferred option into your `docker-compose.yml` file.
+
+!!! note "Windows Users"
+    You must edit the `nakama:volumes:` entry in your `docker-compose.yml` file so that it looks like the following: `/c/Users/<username>/projects/docker:/nakama/data`.
+
+4. Open a Terminal window and navigate to your Nakama directory. For example:
+
+```sh
+cd desktop/nakama
+```
+
+5. To pull all required images and start your application, run the following:
+
+```sh
+docker-compose -f docker-compose.yml up
+```
+
+6. Congratulations! Your Nakama server is now up and running, available at `127.0.0.1:7350`.
+
+![Nakama containers running](images/nakama-compose-running.png)
+
+Use the **Open in Visual Studio Code** button (or that for your IDE) to edit your `docker-compose.yml` file directly.
+
+## Nakama Console
+
+You can also access the Nakama Console by navigating your browser to `localhost:7351`:
```

Comment:
In some OSs, `localhost` acts out and doesn't resolve - it's always better to refer with `127.0.0.1`